97189 AF1 Industry Field

goutamsharmajkt opened the pull request:
Created a new maintenance for the industry and added in the A-F-1 screen

goutamsharmajkt committed:
97189 AF1 Industry Field

Created a new maintenance for the industry and added in the A-F-1 screen

jay-advantzware reviewed (commented):
Need triggers added for the new table?

goutamsharmajkt committed:
97189 AF1 Industry Field

Trigers for new table

rajeshadv committed:
97189 AF1 Industry Field

Files
1. write.trg\customerindustry.p
2. browsers\customerindustry.w
3. create.trg\customerindustry.p
4. delete.trg\customerindustry.p
5. fg\cust-exp.w
6. util\ImportCust.p
7. viewers\customerindustry.w
8. viewers\cust-tot.w
9. windows\customerindustry.w

goutamsharmajkt committed:
97189 AF1 industry field

Made suggested changes

goutamsharmajkt committed:
Merge branch 'feature/97189_AF1_Industry_Field' of https://github.com/Advantzware/Advantzware into feature/97189_AF1_Industry_Field

# Conflicts:
#	Source/browsers/customerindustry.w
#	Source/fg/cust-exp.w
#	Source/util/ImportCust.p
#	Source/viewers/cust-tot.w
#	Source/viewers/customerindustry.w
#	Source/windows/customerindustry.w

goutamsharmajkt committed:
97189 Industry Field

Fixed testing issue

goutamsharmajkt committed:
97189 Industry Field

Made changes suggested by Jay

Commit:
added call to applhelp on F1

If logged in as "nosweat" can create a dynLookup
